prov/sockets/src/sock_fabric.c

```diff
@@ -265,6 +265,7 @@ int sock_verify_info(uint32_t version, struct fi_info *hints)
 	case FI_SOCKADDR_IN:
 		break;
 	default:
+		SOCK_LOG_DBG("Unsupported address format\n");
```

Comment:
I would set this to INFO, rather than debug, so that the message is available in release builds.